### ISSUE TITLE
[Thu] Create UBiosphereProfiler C++ class for inference

### ISSUE THREAD

**Issue by zajalist** [zajalist/GeoForge#43]
Create C++ class UBiosphereProfiler:
- Load ONNX model via NNE
- Extract 33 features from .geoforge data
- Run inference: 33 features -> 17 biome logits
- Return biome prediction per cell
